app/models/fee_structure.rb

```diff
@@ -56,7 +56,7 @@ class FeeCalculation
     validates_numericality_of :amount, greater_than: 0, is_integer: true
     validate :validate_source_is_source_like
 
-    def initialize(**args)
```

Comment:
Ruby 3 has different rules around named and hash arguments at the end of the argument list. This was failing so we switch to the style that Ruby 3 prefers.

We did this in multiple places.